### DISCUSSION TITLE
Setting up a coupled Parent-Metabolite PBPK model

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#880] [Questions & Problems]
Dear OSP community,

I’m extending an already developed PBPK model for a compound that is metabolised by the liver to a major metabolite, in order to predict the PK in healthy volunteers. 

Firstly, I have collected the physiochemical properties of the metabolite, as well as the in vitro measurements that were used to parameterize its clearance process from the literature. Secondly, upon comparing the simulated metabolite exposure vs the observed (experimental) data, I have optimised the hepatic clearance, Lipophilicity (LogP), and the fraction unbound in plasma within the reported literature values.
Below you can see the resulting concentration-time profiles of the parent (in red) and the metabolite (in blue) vs the observed data. A strong misspecification is prominent in the first 30 min of the simulation, even after optimization of relevant parameters within the literature ranges. 

![image](https://user-images.githubusercontent.com/67606842/112535191-46ffe980-8dac-11eb-986c-74eda34c73bb.png)

Then as a trial, I did another parameter identification step with unconstrained values for the previous parameters, resulting in the curve below. While the misspecification is now gone, unrealistic parameter values were obtained, i.e. a LogP more than 10 times below the reported range and the clearance 5 times higher than the reported range.

![image](https://user-images.githubusercontent.com/67606842/112535242-5717c900-8dac-11eb-89bf-d676a386687c.png)

The fact that these unrealistic parameters lead to a good fit led me to the assumption that I am missing some process in between the formation of the metabolite in the liver and its appearance in plasma, has anyone of you had a similar experience and could they point me towards a possible solution? The alternative would be to accept the alternative parameters as such but I am hesitant to do so.

Please let me know if anything is unclear or more information should be provided.
Thanks in advance,
Aya

**Comment by prvmalik** [comment]
Hi Aya

It is likely because the metabolite is being formed too quickly and too extensively. 

Are you attributing 100% of parent clearance to the hepatic route, which forms your major metabolite? Consider the possibility that other clearance mechanisms are at work (minor hepatic metabolite, renal or biliary)

Paul

**Comment by StephanSchaller** [comment]
In addition to @prvmalik comments, consider reduced (from a default of 100cm/min) endothelial permeability (it seems you compound reached liver too quickly).
What is the reported LogP range?
Is the metabolite formed by Phase I or Phase II enzymes?

Best,
Stephan

**Comment by AyaSaleh90** [comment]
Hi Paul, 


> It is likely because the metabolite is being formed too quickly and too extensively.

> Are you attributing 100% of parent clearance to the hepatic route, which forms your major metabolite? Consider the possibility that other clearance mechanisms are at work (minor hepatic metabolite, renal or biliary)
> 
> Paul

I agree with you that the first assumption we made to this curve, is the fast formation of the metabolite and since the parent compound is mainly metabolized by CYP3A4, so the Kcat of CYP3A4 was changed based on the assumption of ~ 90% of the parent is metabolized to the main metabolite (and the other 10% to a sink metabolite).  Then I tried different lower values for this process but it still has this steep peak in the metabolite exposure.

**Comment by AyaSaleh90** [comment]
Hi Stephan, 

> In addition to @prvmalik comments, consider reduced (from a default of 100cm/min) endothelial permeability (it seems you compound reached liver too quickly).

from what I know (as I have little experience in PK-Sim), that is already assumed in the simulation block for each compound,  but How can I change that value?? 

> What is the reported LogP range?

The reported range of the LogP = 2.0 - 2.5 --> for the metabolite
In the first curve, LogP = 2.291 --> optimized using the observed data
While for the second curve, LogP=0.047 

> Is the metabolite formed by Phase I or Phase II enzymes?

The metabolite is formed by CYP3A4 

Best,
Aya

**Comment by StephanSchaller** [comment]
WHat is the LogP for the parent?

**Comment by AyaSaleh90** [comment]
> WHat is the LogP for the parent?

Log P = 2.897 --> previously optimised for the parent from range values (2.8 - 3.53)

**Comment by prvmalik** [comment]
Permeability is probably not restrictive for a compound of logP > 2.5 especially on that short time scale you have presented.

Also check that your metabolite is being formed inside cells rather than in plasma (location of enzyme in the model). 

I see you are implying that there is zero renal and zero biliary clearance?

**Comment by StephanSchaller** [comment]
yes,, I agree with @prvmalik

**Comment by AyaSaleh90** [comment]
Dear Paul,

> Permeability is probably not restrictive for a compound of logP > 2.5 especially on that short time scale you have presented.
> 
> Also check that your metabolite is being formed inside cells rather than in plasma (location of enzyme in the model).
> 
Yes, the enzyme is localized intracellularly in the tissue and endosomal in the vascular endothelium.

> I see you are implying that there is zero renal and zero biliary clearance?

For the metabolite, I only applied total hepatic clearance. While for the parent the GFR = 0.64 which was optimized earlier as well. and no biliary clearance for both.

**Comment by AyaSaleh90** [comment]
Hi everyone, 
I did two other parameter identification steps, the 1st one with the permeability parameter, where the original value was P(plasma <-> interstitial) = 100 cm/min, while the optimised value decreased by almost 100% to give P(plasma<->interstitial) = 0.121 cm/min, resulting in the curve below. And yet the misspecification became more prominent than before (in both early and late phases).
 
![image](https://user-images.githubusercontent.com/67606842/114554035-94160380-9c66-11eb-8d1b-5783310fca8c.png)


The 2nd P.I, with unconstrained values for the following original permeability parameters:

1.	P (plasma<->interstitial) = 100 cm/min
2.	P (interstitial->intracellular) = 0.007268 cm/min  
3.	P (intracellular->interstitial) = 0.007268 cm/min 
 
While the optimised value decreased by ~ 99% for the 3 parameters, to give:

1.	P (plasma<->interstitial) = 1 cm/min 
2. 	P (interstitial->intracellular) = 0.00002714 cm/min 
3.	P (intracellular->interstitial) = 0.00001519 cm/min 
Resulting in the curve below:
 
![image](https://user-images.githubusercontent.com/67606842/114554213-c293de80-9c66-11eb-996e-857c165dd133.png)


The early misspecification is again gone, but I still cannot interpret it physiologically, can someone guide me to what I’m not fully understanding here?
Thanks in advance,
Aya

**Comment by RobinM92** [comment]
Dear colleagues,

Would it be possible that lysosomal trapping is at the basis of this discrepancy? Our formed metabolite is not charged at physiological pH but +/- 40% ionized at pH 5, which could lead to trapping in the lysosomes subsequent slower appearance in the systemic circulation. What would be the most straightforward way to test and/or implement such a hypothesis?

Kind regards,
Robin

**Comment by prvmalik** [comment]
It is rare that a drug is metabolized 100% to one metabolite. As I mentioned before this is likely the source of the error, that your metabolite is being formed too extensively. Lysosomal trapping may occur for drugs with physicochemistry similar to doxorubicin or with high lipophilicity (yours does not have).

**Comment by RobinM92** [comment]
Dear Paul,

Thank you for your quick reply!
In vitro experiments in HLM and hepatocytes confirm >90% metabolism to this one metabolite so it would surprise me if it would be much lower. To decrease the overprediction at Cmax this would mean a stark reduction in the formation of this metabolite which would contradict a lot of other information. Furthermore, the fact that tuning the permeability-related parameters solves the problem seem to point in the direction of some kind of delay in transport from the hepatocyte to the plasma seems to be present.

**Comment by NinaHanke** [comment]
Dear Robin, dear Community, 

To my knowledge, lysosomal trapping is probable for compounds with: 
logP >2 (maybe >1), basic nitrogen with a pKa >8 (maybe >6.5), TPSA <70 Angström² (maybe <100 A²). 
This information is taken from a slide of Andrew Parkinson (Marbach DDI Workshop 2017). 

As a straightforward workaround, you could try to implement a binding partner as a substitute for the trapping and see how it looks. Has somebody tried that? 

Kind regards, Nina

**Comment by AyaSaleh90** [comment]
Dear Nina & OSP community, 

> To my knowledge, lysosomal trapping is probable for compounds with:
> logP >2 (maybe >1), basic nitrogen with a pKa >8 (maybe >6.5), TPSA <70 Angström² (maybe <100 A²).
> This information is taken from a slide of Andrew Parkinson (Marbach DDI Workshop 2017).
> 
Thank you for your suggestion, and I have looked upon the lysosomal trapping from the publication of Andrew Parkinson (https://dmd.aspetjournals.org/content/41/4/897), where Lipophilic drugs (logP > 1)  with ionizable amines (pKa > 6) can accumulate in lysosomes. 

> As a straightforward workaround, you could try to implement a binding partner as a substitute for the trapping and see how it looks. Has somebody tried that?
> 
And, as per your recommendation, I implemented a Dummy binding protein intracellularly with the following assumptions:

![image](https://user-images.githubusercontent.com/67606842/116128790-9bf39000-a6c9-11eb-9db9-9020784ec168.png)
![image](https://user-images.githubusercontent.com/67606842/116128803-a01fad80-a6c9-11eb-96dd-6add329565b6.png)

And below you can see the resulting concentration-time profiles of the parent (in red) and the metabolite (in blue) vs the observed data. The simulation showed the same trajectory as the one produced before with the reduced cellular permeability parameters and the disappearance of the peak in the first 30 min. But I could not solve the clear underprediction of the metabolite.

![image](https://user-images.githubusercontent.com/67606842/116128962-d1987900-a6c9-11eb-9490-d052040deeae.png)
![image](https://user-images.githubusercontent.com/67606842/116128980-d78e5a00-a6c9-11eb-852c-203f4245c35c.png)

So, my question now, what steps shall I take to account for that trapping effect, or what I missed here to give the same trajectory but with lower values?
Regards,

**Comment by NinaHanke** [comment]
Dear Aya, 
As this approach would only be a substitute for lysosomal partitioning, unfortunately there are no literature values for the binding parameters. 
I would try to optimize the reference concentration and the Kd (also try koff) of your binding partner. 
If you have, you could compare the results to your volume of distribution VD, as this should be impacted by binding as well as lysosomal partitioning, and you might get some information from this. 
Kind regards, Nina

**Comment by AyaSaleh90** [comment]
Dear Nina, 

Thanks for your feedback, now I have tried out some optimisation steps for adjusting the reference concentration and the kinetics of the binding partner, and using the optimised values as follow:
1.	Reference concentration = **0.62 umol/L** (instead of **1 umol/L**)
2.	Relative expression of the protein is 100% in the **liver periportal** only (instead of all organs)
3.	Kd = **2.2 nmol/L** (instead of **1 nmol/L**)
4.	Koff = 1 1/min (fixed)
And below you can see the resulting concentration-time profiles of the parent (in red) and the metabolite (in blue) vs the observed data. The simulation has markedly improved, but am I now going in the right direction of explaining this lysosomal trapping or drifting by changing the above-mentioned parameters? 

![image](https://user-images.githubusercontent.com/67606842/116696693-87154600-a9c2-11eb-9b6d-b0cce5d7c32e.png)

![image](https://user-images.githubusercontent.com/67606842/116696719-8e3c5400-a9c2-11eb-9d83-062dd91fa6a7.png)


> If you have, you could compare the results to your volume of distribution VD, as this should be impacted by binding as well as lysosomal partitioning, and you might get some information from this.

And one more question, how can I estimate the Vd/Vss of the metabolite? As far as I know, can I estimate it using the following equation:
Vss= CL * MRT, where the CL is the plasma CL used as input parameter and MRT from the simulated concentration-time profile of the metabolite?

Thanks in advance, 
Aya

**Comment by NinaHanke** [comment]
Dear Aya, 

Your fit looks really nice! Unfortunately, if this is a workaround using binding as a substitute for lysosomal trapping, I think there is no way to physiologically interpret the binding parameter values that resulted from your optimization, or compare them to literature. Unless you actually have binding of your metabolite instead of lysosomal partitioning. Do you know if the metabolite is pharmacologically active? Then it might be binding to the target and you could search for literature binding parameter values (of the metabolite or as an approximation of the parent compound). 

And I think the formula Vss = CL * MRT can only be applied if you have data of your metabolite administered intravenously. But you could assume a similar Vd for your metabolite as measured after iv administration your parent compound. 

Kind regards, Nina
